src/components/main/index.tsx

```diff
@@ -17,36 +17,37 @@ import Loader from "../loader";
 type State = {
   movies: Movie[],
   moviesQuantity: number,
-  loading: boolean
+  reload: boolean,
+  loading: boolean,
+  typeSelected: string,
+  sortBySelected: string
 }
 
+const NUM_PAGE: string = "0";
```

Comment:
The type definition in this case is redundant , TS is smart enough to understand that it's `string`